### PR TITLE
Add dc reliability query

### PR DESCRIPTION
**Pull request by tejohnso**

### CODE REVIEW

**Comment by tejohnso** [comment]
@santiagonoguez @stulees It might be worth adding the module version to the module-down event since we're tracking reliability per version. Otherwise module-down events fora particular day might include multiple versions which won't be accurate.

**Comment by stulees** [comment]
@tejohnso Yes good call on adding module version to module-down event.

**Comment by tejohnso** [comment]
@stulees @santiagonoguez The query now considers watchdog's `module-down` events for the current version of the module. It expects watchdog to supply the version as part of the event_details (eg: licensing2018.01.01.01.01).

**Comment by santiagonoguez** [comment]
@tejohnso which query are you referring to ?

Another option would be to use the existing version column and use that as the module version for module-down ( and maybe module-up ) events.

**Comment by tejohnso** [comment]
@santiagonoguez I had updated the wrong query. It's update now (the query in this PR).

I think it's better to keep *version* consistent to mean the version of the module reporting the current entry rather than changing what *version* means depending on what the event is.

And for module-down / up events, the module name+version is really the detail appropriate for the event.

File: projects/client-side-events/datasets/Module_Events/display_control_reliability.bq [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+recentEventsCurrentVersion AS (
+  SELECT
+    event,
+    display_id,
+    DATE(ts) date
+  FROM
+    `client-side-events.Module_Events.display_control_events`
+  WHERE
+    _PARTITIONTIME < CURRENT_TIMESTAMP()
+    AND _PARTITIONTIME >= TIMESTAMP(DATE_ADD(CURRENT_DATE(), INTERVAL -5 DAY))
```
Comment (stulees) [inline projects/client-side-events/datasets/Module_Events/display_control_reliability.bq]: @tejohnso We have the interval set to 3 days prior for Installer and Player reliability stats. Any reason you're choosing 5? Can we stay consistent with 3?

**Comment by stulees** [comment]
@tejohnso There's no union with an aggregate table that this query will be getting saved to. What is the plan for aggregate table?

**Comment by tejohnso** [comment]
@stulees We're running it every day and manually adding values to a spreadsheet so I didn't think there'd be a need for an aggregate table. If at some point we need one, we  can add as a separate PR, along with possibly combining all these individual module queries into one result table.

**Pull request merged**